readthedocs/core/admin.py

```diff
@@ -94,6 +98,32 @@ def ban_user(self, request, queryset):
 
     ban_user.short_description = 'Ban user'
 
+    def sync_remote_repositories_action(self, request, queryset):
+        formatted_task_urls = []
+
+        for user_id, username in queryset.values_list('id', 'username'):
+            result = sync_remote_repositories.delay(user_id=user_id)
+            job_status_url = reverse(
+                'api_job_status', kwargs={'task_id': result.task_id}
+            )
+            formatted_task_urls.append(
+                format_html(
+                    "<a href='{}'>{} task</a>",
+                    job_status_url,
+                    username
+                )
+            )
+
+        self.message_user(
+            request,
+            mark_safe(
+                'Following sync remote repository tasks were '
+                'triggered: {}'.format(', '.join(formatted_task_urls))
+            )
+        )
```

Comment:
I'm not sure these links are useful/required. @agjohnson what do you think?

Reply:
I just added this to have an easy way to quickly check the task status as I was not sure if there are any internal admin interfaces to check this for each of the triggered tasks by this action.